package.json

```diff
@@ -40,7 +40,10 @@
       "import": "./dist/esm/index.js",
       "require": "./dist/cjs/index.js",
       "default": "./dist/esm/index.js"
-    }
+    },
+    "import": "./dist/esm/index.js",
+    "require": "./dist/cjs/index.js",
+    "default": "./dist/esm/index.js"
```

Comment:
We can probable remove the node-specific exports above given they are exactly the same as these new ones.

Reply:
You're right!